CONTRIBUTING.md

```diff
@@ -146,7 +146,14 @@ mood that describes your changes. (e.g., `Deduplicate the plugins list.` ) See e
 
 ### Preview the documentation
 
-If you make some changes to the `docs/` and you want to preview the build result, simply do:
+PDM docs development requires a few dependencies. Install them as:
+
+```bash
+sudo apt install mkdocs # Or equivalent with the package manager of your choice
```

Comment:
Why do we need to install mkdocs when it is already specified in `pyproject.toml`

Reply:
I agree.

I directly started at https://github.com/pdm-project/pdm/blob/main/CONTRIBUTING.md#preview-the-documentation which threw me the `mkdocs` problem.
I later ran into too many other missing things at which point I realized I missed https://github.com/pdm-project/pdm/blob/main/CONTRIBUTING.md#local-development.

I read `pyproject.toml`. Removed it now.
Please check